formicarium/formicarium/DiffRobot.py

```diff
@@ -6,6 +6,7 @@
 from geometry_msgs.msg import Twist
 from sensor_msgs.msg import LaserScan
 from rclpy.node import Node
+import pygame
```

Comment:
```suggestion
from pygame import Rect
```
dont need to import the complete library only Rect is enough

Reply:
That is correct, it is changed and Rect() is directly called instead of pygame.Rect for the hitbox property